org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizardPage.java

```diff
@@ -122,6 +131,19 @@ public void createControl(Composite parent) {
 		setControl(container);
 		container.setLayout(new GridLayout(4, false));
 
+		new Label(container, SWT.NONE);
+		new Label(container, SWT.NONE);
+		Link preferencesLink = new Link(container, SWT.NONE);
```

Comment:
It's more usual to place links after the content with some spacing, or closer from the field they impact (the templates list here).